crest/Assets/Crest/Crest/Scripts/Helpers/Helpers.cs

```diff
@@ -0,0 +1,19 @@
+// Crest Ocean System
```

Comment:
I'm guessing there isn't already another helpers file that this could go into? If not this is fine.

Adding/removing files can be a pain with assets so just checking.

Reply:
Not that I can see. The plan is to never delete this file even if it becomes empty. It will allow us to add any helper functions that don't warrant a separate file.